packages/website/docs/api.md

```diff
@@ -3,47 +3,49 @@ id: api
 title: Introduction
 ---
 
-The API ref contains details on all the Autocomplete packages.
-
 import Draft from './partials/draft.md'
+import PluginsList from './partials/plugins-list.md'
 
-<Draft />
-
-<!-- - what each package does
-- using react or vue, use this
-- the api ref explains the raw params, but go to the guides for more comprehensive info, etc.
-- package namings, core, recents, presets which provides utils, plugins which are, theme, layouts, etc. -->
+The API reference contains technical information on all Autocomplete packages.
 
-This API references how to import and use the Autocomplete APIs without guidance. Make sure to read the [**Core concepts**](basic-options) first. To learn more about Autocomplete patterns, head over to the [**Guides**](#todo).
+This API references how to import and use the Autocomplete APIs without guidance. Make sure to read the [**Core concepts**](basic-options) first. To learn more about Autocomplete patterns, head over to the [**Guides**](#).
```

Comment:
The link is renamed correctly now:

```suggestion
This API references how to import and use the Autocomplete APIs without guidance. Make sure to read the [**Core concepts**](basic-options) first. To learn more about Autocomplete patterns, head over to the [**Guides**](adding-suggested-searches).
```

Reply:
That's the first guide but not really the entry point. If tomorrow we change what the first guide is, this will become outdated.

Can we maybe create some sort of alias?

Reply:
I wanted to set up Netlify redirects at some point but changed my mind. It can wait a little imo.